acceptance/tests/lookup/v3_config_and_data.rb

```diff
@@ -12,9 +12,13 @@
     confdir = master.puppet('master')['confdir']
     codedir = master.puppet('master')['codedir']
 
+    step "backup global hiera.yaml" do
+      on(master, "cp #{confdir}/hiera.yaml /tmp/C99629-hiera.yaml.bak", :acceptable_exit_codes => [0,1])
```

Comment:
Could you change the backup destination to use `host.tmpfile` so we don't have to assume `/tmp`?